charts/kafka/charts/zookeeper/values.yaml

```diff
@@ -244,6 +246,23 @@ exporters:
       scrapeTimeout: 30s
       scheme: http
 
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+volumePermissions:
+  enabled: true
+  image:
+    registry: docker.io
+    repository: bitnami/minideb
+    tag: buster
```

Comment:
could we use a smaller image, this only seems to be doing a simple chown command, so maybe a busybox image at <1mb size vs this image at ~25mb